src/structures/interfaces/TextBasedChannel.js

```diff
@@ -77,7 +77,7 @@ class TextBasedChannel {
    */
 
   /**
-   * Types of mentions to enable in MessageMentionOptions. It can be one of the following strings:
+   * Types of mentions to enable in MessageMentionOptions. Here are the available strings:
```

Comment:
```suggestion
   * Types of mentions to enable in MessageMentionOptions.
```
Not sure anything additional needs to be said